src/tasks/qa.py

```diff
@@ -172,3 +177,143 @@ def get_metrics(self, reset=False):
             self._score_tracker = collections.defaultdict(list)
 
         return {"ans_f1": ans_f1, "qst_f1": qst_f1, "em": em, "avg": (ans_f1 + em) / 2}
+
+
+@register_task("qasrl", rel_path="QASRL/")
+class QASRLTask(SpanPredictionTask):
+    def __init__(self, path, max_seq_len, name, **kw):
+        """QA-SRL (Question-Answer Driven Semantic Role Labeling)
+        See http://qasrl.org/
+        Move the contents of qasrl-v2 into QASRL
```

Comment:
?

Reply:
Rephrasing to:

```Download, unzip, and rename the "qasrl-v2" folder to "QASRL"```

Reply:
SGTM. Wasn't sure if this was an instruction for the reader or a TODO.